packages/template-retail-react-app/app/hooks/use-einstein.js

```diff
@@ -356,46 +360,67 @@ class EinsteinAPI {
      * https://developer.salesforce.com/docs/commerce/einstein-api/references#einstein-recommendations:Summary
      **/
     async getRecommendations(recommenderName, products, args) {
-        const endpoint = `/personalization/recs/${this.config.siteId}/${recommenderName}`
+        const endpoint = `/personalization/recs/${this.siteId}/${recommenderName}`
         const method = 'POST'
         const body = {
             products: products?.map((product) => this._constructEinsteinProduct(product)),
             ...args
         }
 
-        // Fetch the recommendations
-        const reco = await this.einsteinFetch(endpoint, method, body)
-
-        reco.recommenderName = recommenderName
-
-        return this.fetchRecProductDetails(reco)
+        return this.einsteinFetch(endpoint, method, body)
     }
 
     /**
      * Get a set of recommendations for a zone
      * https://developer.salesforce.com/docs/commerce/einstein-api/references#einstein-recommendations:Summary
      **/
     async getZoneRecommendations(zoneName, products, args) {
-        const endpoint = `/personalization/${this.config.siteId}/zones/${zoneName}/recs`
+        const endpoint = `/personalization/${this.siteId}/zones/${zoneName}/recs`
         const method = 'POST'
 
         const body = {
             products: products?.map((product) => this._constructEinsteinProduct(product)),
             ...args
         }
 
-        // Fetch the recommendations
-        const reco = await this.einsteinFetch(endpoint, method, body)
-
-        return this.fetchRecProductDetails(reco)
+        return this.einsteinFetch(endpoint, method, body)
     }
+}
 
-    async fetchRecProductDetails(reco) {
+const useEinstein = () => {
+    const api = useCommerceApi()
```

Comment:
Why still using this, rather than the new hooks?

Reply:
This is one of the new hook! :)

We could use `useProducts`, but that involves major changes to the useEinstein hook and the consumer components, so I'd like to avoid that in this PR...